spec/liquid_tags/user_subscription_tag_spec.rb

```diff
@@ -46,6 +46,7 @@
     before do
       sign_in subscriber
       visit article_with_user_subscription_tag.path
+      wait_for_javascript
```

Comment:
My first thought when seeing this was that explicit waiting in Capybara tests was always considered to be a bit of an anti-pattern, since Capybara has waiting assertions. So I looked at the `wait_for_javascript` helper, which leads to this [Medium post](https://medium.com/doctolib/hunting-flaky-tests-2-waiting-for-ajax-bd76d79d9ee9). It is indeed where we took this method from, but it's listed under the headline "If you are lazy, blindly wait for ajax". 

I have no problem with merging this, as the flaky specs got quite annoying again recently. But we probably need another concereted effort around improving the quality of our test suite, maybe during the next downtime.

Reply:
Agree, I haven't marked this as ready yet bc I was reproducing successfully but then was not able to but your suggestion has given me another idea to try and work in some waiting assertions.